index.html

```diff
@@ -451,10 +451,10 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
         <b class="header">each</b><code>_.each(list, iteratee, [context])</code>
         <span class="alias">Alias: <b>forEach</b></span>
         <br />
-        Iterates over a <b>list</b> of elements, yielding each in turn to an <b>iteratee</b>
-        function. The <b>iteratee</b> is bound to the <b>context</b> object, if one is
-        passed. Each invocation of <b>iteratee</b> is called with three arguments:
-        <tt>(element, index, list)</tt>. If <b>list</b> is a JavaScript object, <b>iteratee</b>'s
+        Iterates over a <b>list</b> of elements, yielding each in turn to an <a href="#iteratee"><b>iteratee</b></a>
+        function. The <a href="#iteratee"><b>iteratee</b></a> is bound to the <b>context</b> object, if one is
+        passed. Each invocation of <a href="#iteratee"><b>iteratee</b></a> is called with three arguments:
+        <tt>(element, index, list)</tt>. If <b>list</b> is a JavaScript object, <a href="#iteratee"><b>iteratee</b></a>'s
```

Comment:
Not sure we want to have the link duplicated when we have many occurrence in the same section